### PR TITLE
templates: adds missing parts in brief and detailed

### LINKED ISSUE #542: Literature detailed: Author display improvements
Based on the current implementation for inspirehep, author display has to be amended:

https://github.com/inspirehep/inspire/blob/master/bibformat/format_elements/bfe_INSPIRE_authors.py#L27

Important to note the logic about displaying corporate authors and collaborations.

### LINKED ISSUE #597: Brief and detailed production check
Brief and detailed formats still lack some basic information that is currently displayed on production.

This is more obvious in some of the records:

screen shot 2015-12-01 at 5 20 17 pm

The current [brief](https://github.com/inspirehep/inspire/blob/master/bibformat/format_templates/Default_HTML_brief.bft) and [detailed](https://github.com/inspirehep/inspire/blob/master/bibformat/format_templates/Default_HTML_detailed.bft) formats need to be reviewed, and the missing information needs to be added (prior decision if the information is still relevant)

Also the following needs to be fixed:

* [ ] Avoid blind capitalization - titles are being capitalized which does not make sense in most cases (due to acronyms or other parts of the title meant to be in capital letters). Only if a title is all in capital letters (like some older records) we should capitalize the title.

In order to compare one can search any record in inspirehep.net to see how it is currently displayed.

@ioannistsanaktsidis can you have a look?

### PR DESCRIPTION
**Pull request by nikpap**
* Avoid blind capitalization of titles. (closes #597)
* Author display improvements. (closes #542)
* Display report numbers. (addresses #328)
* Displays record information on the right side.

Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>
Co-Authored by: Nick Papoutsis <admin@nils.gr>

### CODE REVIEW

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl [hunk 1]
```diff
@@ -54,6 +62,9 @@
                 {% if record.authors %}
                   <div class="authors">
                     {{ render_record_authors(is_brief=true, show_affiliations=false) }}
+                    {% if record.get('earliest_date') %}
```
Comment (nikpap) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: @ioannistsanaktsidis @jmartinm Is this date supposed to appear next to authors?

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl [hunk 2]
```diff
@@ -28,49 +29,59 @@
       {{ record_collection_heading() }}
     </div>
     <div id="record-title">
-      {{ record_title() }}
+      {{ render_record_title() }}
     </div>
     <div id="record-authors">
       {{ render_record_authors(is_brief=false, number_of_displayed_authors=25) }}
     </div>
+    <div id="record-year">
```
Comment (nikpap) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl]: This date seems kinda out of place(?)
capture d ecran 2015-12-14 a 12 22 44

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl (hunk 1 above)
Comment (ioannistsanaktsidis) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: It is like this in Production. Just placed there to have it. I do not think that it fits there correctly.

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl (hunk 2 above)
Comment (ioannistsanaktsidis) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl]: screen shot 2015-12-14 at 14 39 31

I guess having it here it the best.

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl (hunk 1 above)
Comment (jmartinm) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: Now that you have moved the date to the right side it is not needed there anymore.

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl (hunk 2 above)
Comment (jmartinm) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl]: Yes, at the moment the brief and detailed are not yet consistent.

Brief format:

screen shot 2015-12-15 at 11 26 34 am

Detailed format:

screen shot 2015-12-15 at 11 27 43 am

@nikpap Can you look into a smoother transition between brief and detailed page? We could consider on the detailed page having the same separate section on the right side with the same buttons (and the same labels).

For consistency we could have the same information with the date, and shortcuts to the bottom part of the page (Citations and References).

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl [hunk 3]
```diff
@@ -110,8 +125,9 @@
               {% if record.get('earliest_date') %}
                 <i class="glyphicon glyphicon-calendar"></i> {{ record.get('earliest_date').split('-')[0] }}<br/>
               {% endif %}
-              {% if  record.get('citation_count') > 0  %}
-                <i class="fa fa-quote-left"></i><span><a href="/search?p=refersto:{{ record.get('control_number') }}"> Cited {{ record.get('citation_count') }} times</a></span><br/>
+              {% if record.get('citation_count') > 0  %}
+                <i class="fa fa-quote-left"></i><span><a href="/record/{{ record.get('control_number') }}#citations"  target="_blank"> Cited {{ record.get('citation_count') }} times</a></span><br/>
+>>>>>>> format: HEP detailed record format
```
Comment (jmartinm) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: Merge leftover

File: inspirehep/ext/jinja_filters/record.py [hunk 4]
```diff
@@ -407,3 +407,44 @@ def ads_links(record):
...
+                        pages = ' ' + field['page_artid']
+
+                    out += '<i>' + journal_title + '</i>&nbsp;' +\
+                        journal_volume + year + journal_issue + pages
+                    result.append(out)
+        return result
+
+    @app.template_filter()
+    def is_upper(s):
+        out = False
```
Comment (jmartinm) [inline inspirehep/ext/jinja_filters/record.py]: Can be done in one line with

```python
return s.isupper()
```

File: inspirehep/ext/jinja_filters/record.py [hunk 5]
```diff
@@ -407,3 +407,44 @@ def ads_links(record):
         else:
             link = "%sauthor=%s" % (ADSURL, record['name']['preferred_name'])
         return link
+
+    @app.template_filter()
+    def publication_info(record):
+        result = []
+        if 'publication_info' in record:
+            journal_title, journal_volume, year, journal_issue, pages = \
```
Comment (jmartinm) [inline inspirehep/ext/jinja_filters/record.py]: Another way is:

```python
journal_title = journal_volume = year = journal_issue = pages = ''
```

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl [hunk 6]
```diff
@@ -110,8 +125,8 @@
               {% if record.get('earliest_date') %}
                 <i class="glyphicon glyphicon-calendar"></i> {{ record.get('earliest_date').split('-')[0] }}<br/>
               {% endif %}
-              {% if  record.get('citation_count') > 0  %}
-                <i class="fa fa-quote-left"></i><span><a href="/search?p=refersto:{{ record.get('control_number') }}"> Cited {{ record.get('citation_count') }} times</a></span><br/>
+              {% if record.get('citation_count') > 0  %}
+                <i class="fa fa-quote-left"></i><span><a href="/record/{{ record.get('control_number') }}#citations"  target="_blank"> Cited {{ record.get('citation_count') }} times</a></span><br/>
```
Comment (jmartinm) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: Doing a search is the intended behaviour here. We will keep the functionality as is now - Check clicking on a Citation count here https://inspirehep.net/search?p=author%3Arandall+|+author%3Asundrum+cited%3A300-%3E1350&rm=citation

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl [hunk 7]
```diff
@@ -17,60 +17,93 @@
...
-    </div>
-    <div id="doi-eprint-experiment">
-      {{ record_doi() }}
-      {{ record_arxiv(is_brief=false) }}
-      <div class="cite-pdf-buttons">
-        <div class="btn-group">
-          {{ record_buttons() }}
+    <div class="row">
+        {{ record_cite_modal() }}
+      <div class="col-md-10" id="left-column">
```
Comment (jmartinm) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl]: Do we need an id here or just a class? When naming `ids` we should add some prefix to their name depending on the context where they are. We have seen already some id name clashing.

File: inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl [hunk 8]
```diff
@@ -17,60 +17,93 @@
...
+          {{ render_record_title() }}
+        </div>
+        <div id="record-authors">
+          {{ render_record_authors(is_brief=false, number_of_displayed_authors=25) }}
+        </div>
+        <div class="journal">
+          {{ record_publication_info() }}
+        </div>
+        {% if record.get('report_numbers') %}
+          <div class="report-numbers">
```
Comment (jmartinm) [inline inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl]: This one uses a `class` and the rest like `<div id="external_links"> ` use `ids` would be good to be consistent.

**Comment by jmartinm** [comment]
For the display of report numbers:

screen shot 2016-01-07 at 11 12 52 am

When they are on their own it looks ok but when there are multiple and close to the arXiv number it would be better to display them in one line.

On our user testing sessions the note I have is that the only report number that was helpful in brief was CERN EP report numbers which means Published rather than Conference Proceedings.

The same user mentioned that using badges to show the document type would be very nice and it would make showing even that report number not needed.

@inspirehep/inspire-dir what do you think about the report numbers on brief results for regular users? In the past I remember we discussed the possibility of only showing them if some other basic metadata is not present. Shall we do that? Or remove them altogether once we show a document type badge for each result as they are accessible inside the detailed record.

**Comment by ksachs** [comment]
> only showing them if some other basic metadata is not present.

E.g. in the first example of your screenshot (CMS-PAS) you need the repno to know what it is.

**Comment by kaplun** [comment]
Indeed, for the 4 big experiment, I would tend to do a special treatment and, even when we have a pubnote/DOI, still display the report number, or at least a badge built after the reportnumber prefix.

For example, take these notes from CMS:
http://cds.cern.ch/collection/CMS%20Physics%20Analysis%20Summaries

Here the reportnumbers: `CMS-PAS-FSQ-12-004, CMS-PAS-EXO-15-002, CMS-PAS-HIG-15-010, CMS-PAS-SUS-15-005` carry so much information in so little space (at least for a CMS scientist) that it would be a pity to hide it. We could then take the prefix of such reportnumber and make it in badges such as in: 

`CMS-PAS-FSQ, CMS-PAS-EXO, CMS-PAS-HIG`...

**Comment by ksachs** [comment]
NO - I don't want to single out CERN experiments.
Hide the report number(s) if there is a publication note.

**Comment by jmartinm** [comment]
This branch will need to be rebased and there are some small conflicts to be resolved.

**Comment by jmartinm** [comment]
As I was trying it locally, I fixed the conflicts in my branch https://github.com/jmartinm/inspire-next/tree/655-rebased (in case you want to take it from there to finish the report number part)

**Comment by jmartinm** [comment]
Merged via https://github.com/inspirehep/inspire-next/pull/845

**Pull request closed without merging**